src/classes/keyframe_scaler.py

```diff
@@ -52,19 +53,23 @@ def _update_prop(self, prop: dict):
             keyframes = [prop.get("Points", [])]
         for k in keyframes:
             # Scale the X coordinate (frame #) by the stored factor
-            [point["co"].update({
-                "X": self._scale_x_value(point["co"].get("X", 0.0))
-                })
-                for point in k if "co" in point]
+            if (is_time):
+                log.debug("Updating both values of time keyframes")
```

Comment:
Maybe mention the "X" and "Y" coordinates  vs  "both values", just to be sure the log output is crystal clear